README.md

```diff
@@ -0,0 +1,44 @@
+# TCP Chat server
+
+This project creates a local server and allows users to connect and send chat
+messages through the terminal. When sending messages, you can use various
+commands to send messages to all users or to a specific user.
+
+## How to run
+
+Install any Dependencies from the package.json file into the project root
+directory. Using node, you can run the command ```npm i``` to install all
+depenenedcies.
+
+## Running server
+
+Run the ```server.js``` file using command ```node server.js```. You should see ```PORT:8000 running``` in terminal.
```

Comment:
Quick hint, when you are using backticks that are only on one line, you can use single backticks and it will have the same effect e.g. \`testing\`.